packages/wallet/src/SingleAddressWallet.ts

```
@@ -283,6 +290,21 @@ export class SingleAddressWallet implements ObservableWallet {
       tip$: this.tip$,
       transactionsHistoryStore: stores.transactions
     });
+
+    this.#rollbackSubscription.add(
+      createTransactionReemitter({
+        confirmed$: this.transactions.outgoing.confirmed$,
+        rollback$: this.transactions.rollback$,
+        store: stores.volatileTransactions,
+        submitting$: this.transactions.outgoing.submitting$
+      })
+        .pipe(
+          mergeMap((rollbackTransaction) => from(this.submitTx(rollbackTransaction))),
+          catchError(() => EMPTY) // REVIEW: ignoring errors; already handled as part of the transactions errors
```

Is it ok to ignore the errors?

---

Maybe worth logging as `debug` level?